### ISSUE TITLE
Can't disable auth when using docker image

### ISSUE THREAD

**Issue by conet** [sasanrose/phpredmin#92]
There is no way to unset the default auth using environment variables. This should be available when the service is secured by other means like a reverse proxy.

**Comment by luongvm** [comment]
@conet if you run by one liner you can't do that. but if you use docker-compose or rebuild the docker image mabe you can modify the `config.dist.php` file and remove the `auth` section.

I'm pushing up a PR where you can disable auth by setting the `PHPREDMIN_AUTH_USERNAME` variable to empty.

**Comment by conet** [comment]
Yeah I know that, but I want to be able to do it using environment variables only, rebuilding or mounting files makes a composition less self-contained, plus there is this statement on the main page:
>  Note: As you can see you can use ENV variables to override any configuration directive of PHPRedmin.

**Comment by luongvm** [comment]
@conet please check #93, I just wrote this.

As I inspected, the config would not unset a config node that has children, so `PHPREDMIN_AUTH=` didn't work for me. (Need verification on this).

After #93 is merged you will be able to disable authentication by setting username to empty.

**Comment by conet** [comment]
It looks good, thanks.

**Comment by delagen** [comment]
Use volume mapping to override default config.php and all works (remove auth section)
